abb_experimental.rosinstall

```diff
@@ -0,0 +1,5 @@
+#Temporary until PR accepted by https://github.com/ros-industrial/
+- git: {local-name: abb, uri: 'https://github.com/AustinDeric/abb.git', version: kinetic-devel}
+
+- git: {local-name: industrial_core, uri: 'https://github.com/ros-industrial/industrial_core.git', version: kinetic-devel}
```

Comment:
>@Levi-Armstrong what is your opinion on having industrial_core in the .rosinstall file versus just using the Kinetic release?

@gavanderhoorn I think it would be best to default to using the release version.